apps/io-wallet-support-func/openapi.yaml

```diff
@@ -0,0 +1,220 @@
+      required:
+        - id
+        - is_revoked
+        - created_at
+
+    ProblemJson:
+      type: object
+      properties:
+        type:
+          type: string
+          format: uri
+          description: |-
+            An absolute URI that identifies the problem type. When dereferenced,
+            it SHOULD provide human-readable documentation for the problem type
+            (e.g., using HTML).
+          default: about:blank
+          example: https://example.com/problem/constraint-violation
+        title:
+          type: string
+          description: >-
+            A short, summary of the problem type. Written in english and
+            readable for engineers (usually not suited for non technical stakeholders and not localized); example: Service Unavailable
+        status:
+          type: integer
+          format: int32
+          description: >-
+            The HTTP status code generated by the origin server for this
+            occurrence of the problem.
+          minimum: 100
+          maximum: 600
+          exclusiveMaximum: true
+          example: 200
+        detail:
+          type: string
+          description: |-
+            A human readable explanation specific to this occurrence of the
+            problem.
+          example: There was an error processing the request
+        instance:
+          type: string
+          format: uri
+          description: >-
+            An absolute URI that identifies the specific occurrence of the
+            problem. It may or may not yield further information if
+            dereferenced.
+
+    FiscalCode:
+      type: string
+      description: User's fiscal code.
+      format: FiscalCode
+      x-import: "@pagopa/ts-commons/lib/strings"
+      example: SPNDNL80R13C555X
+
+    GetCurrentWalletInstanceStatusByFiscalCodeBody:
+      type: object
+      properties:
+        fiscal_code:
+          $ref: "#/components/schemas/FiscalCode"
+      required:
+        - fiscal_code
+
+    Timestamp:
+      type: string
+      format: UTCISODateFromString
+      description: A date-time field in ISO-8601 format and UTC timezone.
+      x-import: "@pagopa/ts-commons/lib/dates"
+      example: "2018-10-13T00:00:00.000Z"
+
+    DeviceDetails:
+      type: object
+      description: User device information
+      properties:
+        platform:
+          type: string
+        attestationSecurityLevel:
+          type: number
+        attestationVersion:
+          type: number
+        keymasterSecurityLevel:
+          type: number
+        keymasterVersion:
+          type: number
+        bootPatchLevel:
+          type: string
+        deviceLocked:
+          type: boolean
+        osPatchLevel:
+          type: number
+        osVersion:
+          type: number
+        vendorPatchLevel:
+          type: string
+        verifiedBootState:
+          type: number
+        x509Chain:
+          type: array
+          items:
+            type: string
```

Comment:
The attributes that are optional here are actually required only for Android (`platform==android`). What do you think about using anyOf and distinguishing AndroidDeviceDetails and IosDeviceDetails?

Reply:
done in 24c41e0ad21ff4b37378e8fb9c05ecefe2656c2e